### PR TITLE
Allow each webhook to override it's own channel

### PR DESCRIPTION
**Pull request by rleibman**

### CODE REVIEW

File: files/default/slack_handler_webhook.rb [hunk 1]
```diff
@@ -31,7 +31,6 @@ def initialize(config = {})
     @timeout = @config[:timeout]
     @icon_emoji = @config[:icon_emoji]
     @icon_url = @config[:icon_url]
-    @channel = @config[:channel]
```
Comment (shortdudey123) [inline files/default/slack_handler_webhook.rb:34]: this breaks backwards compatibility

Comment (rleibman) [reply files/default/slack_handler_webhook.rb:34]: Any suggestions? I need per webhook channels.

Comment (shortdudey123) [reply files/default/slack_handler_webhook.rb:34]: keep this in there and use this option if a webhook specific one is not given

**Comment by martinb3** [comment]
Hello! I'd agree with @shortdudey123 that this _should_ be a backwards-compatible change / feature enhancement. We should be able to support both the global webhook and a per-channel webhook, without breaking one another.

Also, please don't bump the version in your PR. We can do that on release.

**Pull request merged**